packages/flag-evaluation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bucketco/flag-evaluation",
-  "version": "0.0.7",
+  "version": "1.0.0",
```

Comment:
let's keep at `0.x.x`.. In semver, different rules apply below the big 1.0 and I'd like to signal that this library isn't for general consumption. Lets make this `0.1.0`